docs/whats-new/staging-release-notes.md

```diff
@@ -1,8 +1,23 @@
 # Staging release notes
 
-## Aug 2023
+## September 2023
 
-**16 Aug 2023**
+**13 September 2023**
+
+Frontend version: 1.0.0-20230911.0925 | Backend version: 1.69.0-20230908.1609
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Improved tenant user management UI | We have introduced a new user management interface. This interface now enables tenants to utilise the search function and filter based on account type and user status.|
+| **Change** | Separation of user and product accounts | We have separated *user* and *product* accounts. Within the new UI, there are now distinct tabs for *user account* and *product account* types. To align our automation API with this change, we have updated it to provide user and product account types via separate API calls.|
+| **Change** | *accountType* Property in GetGroup API | We have added the *accountType* property for users/owners in the *Get Group* API.|
+| **Fix** | Eliminated duplicated webhook events | We have addressed and resolved an issue related to duplicate webhook events. Previously, some users were experiencing the problem of receiving duplicate events through webhooks. |
+| **Fix** |  Resolved infinite loop in email service | We have resolved an issue related to our email service. Previously, the email service was encountering an infinite loop when it received an invalid target group, causing emails to cease being sent after a certain period. |
```

Comment:
@BananZG `causing emails to cease being sent after a certain period.` <-- this sounds like emails were being sent initially then ceased to be sent eventually, is this correct? or emails are not being sent at all in the first place?